packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -55,6 +55,11 @@ export class VaTextInput {
    */
   @Prop() error?: string;
 
+  /**
+   * Whether or not to add usa-input--error as class if error message is outside of component
+   */
+  @Prop() reflectInputError?: boolean = false;
```

Comment:
Thank you for adding this! Just for future us to remember why we did this, it was added here the same way that it was in `va-select` which was required to trigger the error state for the children components: https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-select/va-select.tsx#L61-L64